tests/fixtures/update-deps.js

```diff
@@ -1,14 +1,58 @@
-module.exports = [ {
-	name:'update-deps PR',
-	workflow:'update-deps.yml',
-	branch: 'master',
-	config: {
-		'updateDeps': {
+let date = new Date();
+let testNpmCheck = true;
+let npmCheckDay = testNpmCheck ? date.getDate() : date.getDate() + 1;
+
+module.exports = [
+	{
+		name: 'update-deps branch without package.json file',
+		workflow: 'update-deps.yml',
+		branch: 'master',
+		config: {
 			'targetBranch': 'master'
 		}
-	},
-	fileList: [ {
-		src: 'deps-package.json',
-		dest: 'package.json'
-	} ]
-} ];
+	}, {
+		name: 'update-deps on up-to-date branch',
+		workflow: 'update-deps.yml',
+		branch: 'master',
+		config: {
+			'targetBranch': 'master'
+		},
+		fileList: [ {
+			src: 'deps-package-up-to-date.json',
+			dest: 'package.json'
+		} ]
```

Comment:
There is an issue with this test that after a while the packages (see https://github.com/ckeditor/ckeditor4-workflows-common/pull/28/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.yml#diff-aec01d3969cc571603b309d74638b18fd1eff39fc547aa979fc6b27c9dc0e86cR11) may not be up to date :thinking: So we probably need to have previous major latest minor like `^2.6.12` for core-js, since `^3.13.0` will cause an update already (checked with https://semver.npmjs.com/): 

![](https://user-images.githubusercontent.com/1061942/121205096-acc22480-c877-11eb-86ad-32e7034b7ecf.png)

Reply:
Also, even if we use `^2.6.12`, npm-check will update it anyway 🤔 So, two workflows for `npm-check` and `npm update` looks even more valid.